development/language/usage.rst

```diff
@@ -0,0 +1,79 @@
+=========================
+Using the Language System
+=========================
+
+phpBB3 uses a language system to guarantee easy localisation. All strings presented to the user are first looked up into an internal dictionary. Thus it is also a requirement for MODs to use said system for all output presented to the user. This chapter will show you how to use the language system and how to add your custom entries.
+
+
+Using the language System in php
+================================
+
+The object holding the language dictionary for the current user is - hardly surprising - the ``$user`` object. We generally refer to the name of the word in the dictionary as **language key**, to the returned value as **language string** or **language value**. Both together are a **language entry**. By convention, the language key should always be all-uppercase. Language entries are collected in language files.
+
+To lookup a language entry inside of phpBB, query the ``$user`` object like this: ``$user->lang('LANG_KEY')``, which will return the language string assigned to the key ``LANG_KEY`` in the current user's language. Note that many functions in phpBB automatically translate their arguments; passing the language key as argument is usually enough.
+
+The dictionary is not held ready all at once, but is split up in several files, which have to be manually loaded on demand. The files are stored inside the language directory, named after its `language tag <https://area51.phpbb.com/docs/30x/coding-guidelines.html#translation>`_, where one subdirectory is used for each installed language. The default language files are in the root of that subdirectory, while ACP language files go into the /acp subdirectory; files for MODs should be placed in the mods subdirectory. Finally, email templates are placed in the email subdirectory.
```

Comment:
The dictionary is not held ready all at once => The dictionary is not loaded into memory all at once

MODs again?
Also, subdirectory structure in the language directory is not this strict anymore for extensions.